napari/util/misc.py

```diff
@@ -210,3 +211,23 @@ def segment_normal_vector(a, b):
     else:
         unit_norm = normal/norm
     return unit_norm
+
+
+class StringEnum(Enum):
+    def _generate_next_value_(name, start, count, last_values):
+        """ autonaming function assigns each value its own name as a value
+        """
+        return name
+
+    @classmethod
```

Comment:
this doesn't need to be a classmethod from what i can tell in the docs

Reply:
Good point. I changed `_missing_()` to an instance method.